lib_spellcheck.gyp

```diff
@@ -64,6 +64,10 @@
       'include_dirs': [
         '/usr/local/macold/include/c++/v1',
       ],
+    }], [ 'build_linux', {
+      'cflags_cc': [
+        '<!(pkg-config --cflags enchant)',
+      ],
     }]],
   }],
-}
\ No newline at end of file
+}
```

Comment:
Nitpick: please remove this change.

Reply:
Well, having an EOL at end of each file is a good practice (I think so) that is quite common in Unix world.

Reply:
I don't mind to have an EOL at end of each file.
But such changes are not relevant to the point of current commit.